components/jenkins/cosmosdb.tf

```diff
@@ -42,6 +42,10 @@ resource "azurerm_cosmosdb_account" "cosmosdb" {
     location          = "UK West"
     failover_priority = 1
   }
+  identity {
+    type         = "UserAssigned"
+    identity_ids = [data.azurerm_user_assigned_identity.monitoring_mi.id]
+  }
```

Comment:
Is this needed?

Reply:
yes it is to assign it to the cosmosdb otherwise it wont tell it to use it

Reply:
isn't that being done by? https://github.com/hmcts/cft-jenkins-infrastructure/pull/44/files#diff-9a0a0ae21fa40a61cf8301f886802370082e693f5752f7327f8314f28cbcfb8dR104
This would be assigning only one mi to the db in a different way

Reply:
you need both as far as I know. You need to assign it and give it permission but can remove it and see first

Reply:
I can't see anywhere else we've done this to suggest we need it - do you have an example?

You'd just be giving one monitoring-mi in production assignment

Reply:
/ not sure what this identity assignment aims to do, especially with a list of just one MI